src/main/java/frc/robot/commands/drivetrain/ResetGyro.java

```diff
@@ -30,6 +32,7 @@ public void initialize() {
     @Override
     public boolean isFinished() {
         return (Timer.getFPGATimestamp() - startTime) >= delay;
+        //TODO: make it return true after testing
```

Comment:
You created the timeoutCounter but you didn't actually use it.

Reply:
I have made these changes.